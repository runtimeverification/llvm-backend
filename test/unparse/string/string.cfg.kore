// RUN: %kprint-check
Lbl'-LT-'generatedTop'-GT-'{}(Lbl'-LT-'k'-GT-'{}(kseq{}(inj{SortString{}, SortKItem{}}(\dv{SortString{}}("hello world!")),dotk{}())),Lbl'-LT-'generatedCounter'-GT-'{}(\dv{SortInt{}}("0")))
