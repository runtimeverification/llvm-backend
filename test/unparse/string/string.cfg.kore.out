<k>[0m
  "hello world!" ~> .K
</k>[0m
